spec/linter-stylelint-spec.js

```diff
@@ -301,4 +303,33 @@ describe('The stylelint provider for Linter', () => {
       );
     });
   });
+
+  describe('works with JS files (language-babel) with Styled Components and', () => {
+    beforeEach(() => {
+      atom.config.set('linter-stylelint.disableWhenNoConfig', false);
```

Comment:
There _is_ a configuration for those files, is there any reason you are disabling this setting?